### PR TITLE
Amended changes for a motion of no confidence

### PR DESCRIPTION
**Pull request by ekohilas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -334,4 +337,16 @@ UNSW CSESoc's Constitution; Easily track changes and see who suggested what amen
...
         8.9.1 Each candidate must reach the quota of votes as determined by the Droop quota for that position.
         8.9.2 When electing Co-presidents, all first and second preferences shall be counted as first preferences.
-
+    8.10 Upon pronouncement of the election results, but before the Handover Meeting, the outgoing Executives may choose
+         to hold a motion of no confidence against any Executive-elects whereby a two thirds majority vote by the outgoing
+         Executives will overrule the election results.
+         8.10.1 For such a motion to be held:
+                8.10.1.1 The motion must be declared publicly to the society
+                8.10.1.2 The reasons for the motion must abide to the definition of the motion of no confidence.
+                8.10.1.3	The outgoing executives must produce a publicly available document to the Society website,
```
Comment (jake-waratahs) [inline README.md]: nit: indent

**Comment by abethcrane** [comment]
Couple of thoughts (feel free to take or leave them):

- Vote of No Confidence doesn't seem like the right term for "previous exec think new exec won't be good".
- Why should this happen after results are announced? Seems like it'd be better to just reject somebody's application to be in the election than to let them win and then reject them immediately.

**Comment by nicolargh** [comment]
Motion defeated at the 2017 AGM.

**Comment by ekohilas** [comment]
Thank you both for you feedback :)
I was unaware of multiple things while writing this that were then clarified during the AGM.

**Pull request closed without merging**